<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="5CG4503MCF" name="com.ericsson.NDOrnd.dopuatautomation.testcases.VDT_TraceInputValidationTest" tests="3" failures="0" timestamp="18 Sep 2019 08:14:36 GMT" time="1841.509" errors="1">
  <testcase name="VerifyTIVErrorNotificationAlertPopUp" time="0.079" classname="com.ericsson.NDOrnd.dopuatautomation.testcases.VDT_TraceInputValidationTest"/>
  <testcase name="VerifyChartValuesWithDateSelected" time="1045.741" classname="com.ericsson.NDOrnd.dopuatautomation.testcases.VDT_TraceInputValidationTest"/>
  <testcase name="VerifyTableWithCellFiltering" time="795.689" classname="com.ericsson.NDOrnd.dopuatautomation.testcases.VDT_TraceInputValidationTest">
    <error type="java.lang.StringIndexOutOfBoundsException" message="String index out of range: 5">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.ericsson.NDOrnd.dopuatautomation.pages.VDT_TraceInputValidationPage.VerifyTableWithCellFilterRopLevel(VDT_TraceInputValidationPage.java:1374)
	at com.ericsson.NDOrnd.dopuatautomation.testcases.VDT_TraceInputValidationTest.VerifyTableWithCellFiltering(VDT_TraceInputValidationTest.java:184)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- VerifyTableWithCellFiltering -->
</testsuite> <!-- com.ericsson.NDOrnd.dopuatautomation.testcases.VDT_TraceInputValidationTest -->
